win32ss/gdi/eng/lineto.c

```diff
@@ -78,7 +78,7 @@ HandleStyles(
     else
     {
         iStyle = 0;
-        lStyleMax = MAX_COORD;
+        lStyleMax = MAXLONG;
```

Comment:
OK I would then suggest adding `ASSERT(cStyles);` in these (previously modified) code blocks, where quantities are being evalutated `% cStyles` .

Reply:
ASSERT's added. Thanks.